### DISCUSSION TITLE
Getting 401 error in chrome console when connecting to couchdb but works normally with firefox

### DISCUSSION THREAD

**Discussion by prog7707** [apache/couchdb#4773] [Q&A]
I am trying to connect a webapp to my couchdb server that is hosted on a secure domain https://mydomain.com.
My cors config:
[cors]
credentials = true
origins = *
headers = X-Couch-Id, X-Couch-Rev
methods = GET,POST,PUT,DELETE,OPTIONS
max_age = 3600
[couch_httpd_auth]
same_site = lax 
allow_persistent_cookies = true

-------------------------------
when i put same_site=none , i cant login to https://mydomain.com/_utils but changing it to lax fixed that.
But still the app is unable to login to couchdb when using chrome, i tried serveral chrome on different pc and got same error. And with Firefox everything works normally.
In nginx i have these config:
server {
listen 80 default_server;
 server_name mydomain.com;
location / {
proxy_pass http://localhost:5984;
proxy_redirect off;
proxy_set_header Host $host;
proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
proxy_set_header X-Forwarded-Proto $scheme;

if ($request_method = 'OPTIONS') {
        add_header 'Access-Control-Allow-Origin' 'https://web.myapp.app'; # Set to your application's origin
        add_header 'Access-Control-Allow-Methods' 'GET, POST, PUT, DELETE, OPTIONS';
        add_header 'Access-Control-Allow-Headers' 'Origin, X-Requested-With, Content-Type, Accept, Authorization';
        add_header 'Access-Control-Allow-Credentials' 'true'; # Allow credentials
        add_header 'Access-Control-Max-Age' 1728000;
        add_header 'Content-Type' 'text/plain charset=UTF-8';
        add_header 'Content-Length' 0;
        return 204;
    }

}

    listen 443 ssl; # managed by Certbot
    ssl_certificate /etc/letsencrypt/live/mydomain.com/fullchain.pem; # managed by Certbot
    ssl_certificate_key /etc/letsencrypt/live/mydomain.com/privkey.pem; # managed by Certbot
    include /etc/letsencrypt/options-ssl-nginx.conf; # managed by Certbot
    ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot

}
-----------------------
can any one help please.

**Comment by rnewson** [comment]
Try setting [cookie_domain](https://docs.couchdb.org/en/stable/config/auth.html#chttpd_auth/cookie_domain)?

**Comment by prog7707** [comment]
I just tried this and aame issue. It didnt work.
